extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -47,6 +47,11 @@ export const RTC_VENDORS = {
     macros: ['PLACEMENT_ID'],
     disableKeyAppend: true,
   },
+  indexexchange: {
+    url: 'https://amp.casalemedia.com/amprtc?v=1&p=HREF&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&s=SITE_ID',
```

Comment:
Recommend moving HREF macro to the end of the url given we have a url limit after which we truncate

Reply:
Moved the `HREF` macro to the end of the URL in commit 23fab56637d9e88de5322967215b8b1a5ecb0217.